elasticsearch-vmss/azuredeploy.json

```diff
@@ -0,0 +1,504 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "clusterName": {
+      "type": "string",
+      "defaultValue": "es-azure",
+      "metadata": {
+        "description": "The name of the cluster."
+      }
+    },
+    "vmSku": {
```

Comment:
It doesn't need to be a param - using a var with a sku that works in all clouds will work just fine.  If someone takes the template to prod, they can parameterize as needed (same as other samples)